test/tektonbundles_test.go

```diff
@@ -515,3 +383,148 @@ func TestTektonBundlesUsingImproperFormat(t *testing.T) {
 		t.Fatalf("Error waiting for PipelineRun to finish with expected error: %s", err)
 	}
 }
+
+func tarImageInOCIFormat(namespace string, img v1.Image) ([]byte, error) {
```

Comment:
Are just these just moved around? or modified?

Reply:
yeah those are just moved at the end of the file (to have the Test at first).. cosmetic change :sweat_smile: